package.json

```diff
@@ -30,7 +30,8 @@
     "bulma": "^1.0.2",
     "onscan.js": "^1.5.2",
     "open-library-api": "^0.1.2",
-    "openapi-fetch": "^0.12.2"
+    "openapi-fetch": "^0.12.2",
+    "uuid": "^10.0.0"
```

Comment:
might be possible to remove in favor of https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID

I'm not :100: sure, but would be neat if true.